packages/valory/protocols/ledger_api/README.md

```diff
@@ -71,13 +73,13 @@ reply:
   get_state: [state, error]
   state: []
   get_raw_transaction: [raw_transaction, error]
-  raw_transaction: [send_signed_transaction]
+  raw_transaction: []
   send_signed_transaction: [transaction_digest, error]
-  transaction_digest: [get_transaction_receipt]
+  transaction_digest: []
   get_transaction_receipt: [transaction_receipt, error]
   transaction_receipt: []
   error: []
-termination: [balance, state, transaction_receipt, error]
+termination: [balance, state, raw_transaction, transaction_digest, transaction_receipt, error]
```

Comment:
This is causing some test failures

Reply:
Failures for ledger connection will be fixed once I port connection from autonomy repo, but there are also failures on erc1155_deploy skill

Reply:
We can ignore test failures on `erc1155_deploy` for now and open an issue to fix later.